### DISCUSSION TITLE
Documentation including page with redocusaurus yaml

### DISCUSSION THREAD

**Discussion by rogatec** [rohit-gohri/redocusaurus#70] [Q&A] [answered]
Hey @rohit-gohri ,

is it possible to use an existing documentation with docusaurus and adding a new sidebar page, which references to a redocusaurus yaml file?

I tried it with the given documentation, but I'm always getting a WebPacket error.
```bash
WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.resolve has an unknown property 'fallback'.
```

Any thoughts?

**Comment by rohit-gohri** [comment, marked as answer]
Are you on the latest docusaurus version? This looks like you are still on "v2.0.0-alpha.x", you should upgrade to the latest version. If you could post your package.json dependencies, I might be able to get a better idea of the problem

**Reply by rogatec** [reply]
Hey you are right, I wasn't and this fixed it. Thanks for the hint
